xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/main/java/org/xwiki/rendering/transformation/macro/RawBlockFilter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rendering.transformation.macro;
+
+import org.xwiki.component.annotation.Role;
+import org.xwiki.rendering.block.RawBlock;
+import org.xwiki.rendering.macro.MacroExecutionException;
+
+/**
+ * Filtering for raw blocks to clean and sanitize raw content.
+ *
+ * @version $Id$
+ * @since 14.6RC1
```

Comment:
The target will probably be 14.7RC1?

Reply:
Yes, I originally targeted 14.6 but I fear that's too risky.

Reply:
With the bank holidays, we are virtually the day before the delivery in terms of working days. So definitely +1 to postpone this for the next RC.